package.json

```diff
@@ -9,6 +9,7 @@
     "diff": "^2.1.1",
     "graceful-fs": "^4.1.3",
     "istanbul": "^0.4.2",
+    "jest-mock": "^0.0.2",
```

Comment:
this branch probably won't build until `jest-mock` is released. I'm not sure what the release flow is